### PR TITLE
rework of NCL-732 - use config props in auth module

### PR DESCRIPTION
**Pull request by pslegr**
Conflicts:
	integration-test/src/test/java/org/jboss/pnc/auth/ExternalAuthFacadeTest.java

rework propertis loading from json instead of System.getEnv()

NCL-732

### CODE REVIEW

File: common/src/main/java/org/jboss/pnc/common/json/ConfigurationJSONParser.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
             }
             throw new ConfigurationParseException("Config could not be parsed");
         } catch (Exception e) {
+            e.printStackTrace();
```
Comment (matejonnet) [inline common/src/main/java/org/jboss/pnc/common/json/ConfigurationJSONParser.java]: Any particular reason to not use logging ?

Comment (pslegr) [inline common/src/main/java/org/jboss/pnc/common/json/ConfigurationJSONParser.java]: thx !
will fix it

**Comment by pslegr** [comment]
retest this please

**Comment by pslegr** [comment]
need rebase, created https://github.com/project-ncl/pnc/pull/296 instead

**Pull request closed without merging**